### PR TITLE
docs: timely build

### PR DESCRIPTION
**Pull request by wingkwong**
## Change Summary

- add documentation for timely build (ref: #2447)

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [x] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

### CODE REVIEW

**Comment by o1lab** [comment]
@wingkwong. : Thanks for preemptively add this :)

Have asked @pranavxc - to make the command to `brew tap nocodb/nocodb`. So pls update

**Comment by wingkwong** [comment]
@o1lab I added it here as it doesn't like it's for timely-build. @pranavxc Please correct me if I'm wrong.

image

**Comment by pranavxc** [comment]
> @o1lab I added it here as it doesn't like it's for timely-build. @pranavxc Please correct me if I'm wrong.
> 
> image

Yes, Homebrew is only for main release.

File: packages/noco-docs/content/en/getting-started/installation.md [hunk 1]
```diff
@@ -57,6 +57,14 @@ npm install
 npm start
 ```
 
+### Homebrew
+
+```bash
```
Comment (pranavxc) [inline packages/noco-docs/content/en/getting-started/installation.md:62]: Replace it with

```bash
```bash
brew tap nocodb/nocodb
brew install nocodb
nocodb
```

**Review by pranavxc — CHANGES_REQUESTED** [review]

File: packages/noco-docs/content/en/getting-started/installation.md (hunk 1 above)
Comment (wingkwong) [reply packages/noco-docs/content/en/getting-started/installation.md:62]: Updated

**Review by pranavxc — APPROVED** [review]

**Pull request merged**